commands.txt

```diff
@@ -7,8 +7,10 @@ SONAR_TOKEN=`$SD_CURL_CMD_WRAPPER "curl -H \"Authorization: Bearer $SD_TOKEN\" $
 VERSION=`cat sonar-project.properties | awk -F= '/sonar.projectVersion=/{print $2}'`
 PROJECT_VERSION=${VERSION:='not provided'}
 SONAR_OPTIONS=`eval echo "$SD_SONAR_OPTS"`
+if [ "$SD_SONAR_ENTERPRISE" = false ]; then SD_SONAR_PROJECT_KEY=`job:$SD_JOB_ID`; else SD_SONAR_PROJECT_KEY=`pipeline:$SD_PIPELINE_ID`; fi
```

Comment:
for enterprise edition, we should allow jobId as project key as well, when overriding via env variable. will this be added in a new PR?

Reply:
yeah that would be second pass